versions/3.0.md

```diff
@@ -1124,6 +1100,185 @@ application/json:
 
 ```
 
+##### Considerations for file uploads
+
+In contrast with the 2.0 specification, describing `file` input/output content in OpenAPI is 
+described with the same semantics as any other schema type. Specifically:
+
+```yaml
+# content transferred with base64 encoding
+schema:
+  type: string
+  format: base64
+
+# content transfered in binary (octet-stream):
+schema:
+  type: string
+  format: binary
+
+# multiple files:
+schema:
+  type: array
+  items:
+    type: string
+    format: binary
+```
```

Comment:
I'm not sure this is true.  See http://stackoverflow.com/questions/913626/what-should-a-multipart-http-request-with-multiple-files-look-like